tests/_support/Traits/With_Uopz.php

```diff
@@ -104,4 +104,22 @@ private function set_class_fn_return( $class, $method, $value, $execute = false
 		uopz_set_return( $class, $method, $value, $execute );
 		$this->uopz_set_returns[] = [ $class, $method ];
 	}
+
+	private function set_class_property( $object, $field, $value ) {
+		if ( ! function_exists( 'uopz_set_property' ) ) {
+			$this->markTestSkipped( 'uopz extension is not installed' );
+		}
+		uopz_set_property( $object, $field, $value );
```

Comment:
The trait should define a method to run `@after` (see examples in this trait and others) to unset any still applicable property.